03-algorithms-sorting/barChart.html

```diff
@@ -0,0 +1,43 @@
+<html>
+  <head>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
+      google.charts.load('current', {packages:['corechart']});
```

Comment:
Thank you. I wonder if I can send the data from Ruby to html. I manually typed the Benchmark result in the html. It doesn't seem efficient... :grimacing:

```html
 ["Insertion Sort", 9.74670983850956e-05],
 ["Selection Sort", 0.00021768081933259964],
 ["Bubble Sort", 5.0709815695881844e-05],
 ["Merge Sort", 0.0001670608762651682],
 ["Quick Sort", 0.0003114859573543072],
 ["Heap Sort", 0.00019613001495599747],
 ['Bucket Sort', 8.90891533344984e-05]
```